components/public-api/gitpod/v1/user.proto

```diff
@@ -11,13 +11,69 @@ option go_package = "github.com/gitpod-io/gitpod/components/public-api/go/v1";
 service UserService {
   // GetAuthenticatedUser allows to retrieve the current user.
   rpc GetAuthenticatedUser(GetAuthenticatedUserRequest) returns (GetAuthenticatedUserResponse) {}
+
+  // UpdateUser updates the properties of a user.
+  rpc UpdateUser(UpdateUserRequest) returns (UpdateUserResponse) {}
+
+  // SetWorkspaceAutoStartOptions updates the auto start options for the Gitpod Dashboard.
+  // +internal - only used by the Gitpod Dashboard.
+  rpc SetWorkspaceAutoStartOptions(SetWorkspaceAutoStartOptionsRequest) returns (SetWorkspaceAutoStartOptionsResponse) {}
```

Comment:
Do we really need this method? I thought autoStart option is not useful, and can cause some confuse for me (Once saw that we want to remove it, I'm happy with that Closed PR XD, but closed already).

Reply:
I'm not aware of removing auto start. Do we have an issue or any discussion on that?

Reply:
Sorry @AlexTugarev , this is not-blocking comment.

[This is the PR](https://github.com/gitpod-io/gitpod/pull/18645) to remove auth start option. I'm not clear if we want to keep it or remove it.

Reply:
It's still used to preserve the options for the next workspace start.

Reply:
Also, there is the `autostart` options to the prefixed URL.